commands/box/contract/contract.go

```diff
@@ -510,6 +464,36 @@ func send(cmd *cobra.Command, args []string) {
 		fmt.Println(cmd.Use)
 		return
 	}
+	//default abi
+	_, err := ioutil.ReadFile(abiIdxFile)
+	if err != nil {
+		fmt.Println("Do you want to use official contract? [Y/n] ")
+		input, err := terminal.ReadPassword(int(os.Stdin.Fd()))
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+		switch string(input) {
+		case "n":
+		case "N":
+		default:
+			index, err := attainIndexFromFile("contracts/bonus.abi", "the file of default abi", true)
```

Comment:
write a function to encapsulate these code to implement find original contract function